mvpa2/measures/rsa.py

```diff
@@ -215,8 +221,10 @@ def __init__(self, target_dsm, **kwargs):
         # init base classes first
         Measure.__init__(self, **kwargs)
         self.target_dsm = target_dsm
+        self.control_dsms = control_dsms
         if self.params.comparison_metric == 'spearman':
             self.target_dsm = rankdata(target_dsm)
+            if control_dsms != None: self.control_dsms = [rankdata(dm) for dm in control_dsms]
```

Comment:
correct way to check for None (or not) (here and below) is `control_dsms is not None`